Zend/zend_vm_def.h

```diff
@@ -1248,6 +1250,7 @@ ZEND_VM_HELPER(zend_post_incdec_property_helper, VAR|UNUSED|CV, CONST|TMPVAR|CV,
 		/* here we are sure we are dealing with an object */
 
 		if (EXPECTED(Z_OBJ_HT_P(object)->get_property_ptr_ptr)
+			&& EXPECTED(!ZEND_CLASS_HAS_TYPE_HINTS(Z_OBJCE_P(object)))
```

Comment:
This makes all ++/-- operations on properties of class with typed properties slower. Even if the property doesn't have type.